src/refactorings/simplify-ternary/index.ts

```diff
@@ -10,7 +14,10 @@ const config: RefactoringWithActionProvider = {
   },
   actionProvider: {
     message: "Simplify ternary",
-    canPerform: hasTernaryToSimplify
+    createVisitor: createTernaryToSimplifyVisitor,
+    updateMessage(path: t.NodePath): string {
+      return `Simplify ternary (for demo purposes only: ${path.node.type})`;
```

Comment:
![image](https://user-images.githubusercontent.com/1094774/76958466-2a55e980-68ee-11ea-9123-dfbc4ea5ab31.png)

I can see it's working 👍 

I'll update the message so it's OK to deploy once on master. Good job 👍